github/pulls_reviews.go

```diff
@@ -0,0 +1,14 @@
+package github
+
+import "time"
+
+// No docs are out for this quite yet.
+type PullRequestReview struct {
+	ID          *int       `json:"id,omitempty"`
+	User        *User      `json:"user,omitempty"`
+	Body        *string    `json:"body,omitempty"`
+	SubmittedAt *time.Time `json:"submitted_at,omitempty"`
+
+	// State can be "approved", "rejected", or "commented".
+	State *string `json:"state,omitempty"`
+}
```

Comment:
Suggestion. This single struct doesn't really deserve an entirely new file. How do you feel about moving it under `PullRequestReviewEvent`, which is the first (and currently only) place it's used?

If it ever becomes used in many other places and there's much more review related code, then moving it into a new separate file would make sense. It seems too early to do that now.